### PR TITLE
Fr/feature/tools block

### PR DESCRIPTION
**Pull request by Kragov**

### CODE REVIEW

File: docs/src/content/home.ts [hunk 1]
```diff
@@ -0,0 +1,262 @@
+import GitHubLogo from "../assets/socials/gh-logo";
+import TwitterLogo from "../assets/socials/tw-logo";
+import DiscordLogo from "../assets/socials/dc-logo";
+import VictorImage from "../assets/images/victor.png";
+import KyberImage from "../assets/images/trustedTeamsLogos/kyber.png";
+import JustimImage from "../assets/images/justin.jpg";
```
Comment (usulpro) [inline docs/src/content/home.ts]: typo

File: docs/src/components/ui/DesktopMenu.tsx [hunk 2]
```diff
@@ -144,5 +143,5 @@ export default Menu;
 
 Menu.defaultProps = {
```
Comment (usulpro) [inline docs/src/components/ui/DesktopMenu.tsx]: UI components should only receive their props from parent components

File: docs/src/content/home.ts [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+import SolidityImageMobile from "../assets/feature-cards/Mobile/SolidityImage.png";
+import SolidityImageDesktop from "../assets/feature-cards/Desktop/SolidityImage.png";
+import FlexibilityImageMobile from "../assets/feature-cards/Mobile/FlexibilityImage.png";
+import FlexibilityImageDesktop from "../assets/feature-cards/Desktop/FlexibilityImage.png";
+import ExtensibleImageMobile from "../assets/feature-cards/Mobile/ExtensibleImage.png";
+import ExtensibleImageDesktop from "../assets/feature-cards/Desktop/ExtensibleImage.png";
+import FastIterationImageMobile from "../assets/feature-cards/Mobile/FastIterationImage.png";
+import FastIterationImageDesktop from "../assets/feature-cards/Desktop/FastIterationImage.png";
+
+const defaultSocialsItems = [
```
Comment (usulpro) [inline docs/src/content/home.ts]: social items are common for the entire website - so please move them to config file

File: docs/src/content/home.ts [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+const heroBlockContent = {
+  title: "Ethereum development environment for professionals",
+  tagline: "Flexible. Extensible. Fast.",
+  cta: {
+    title: "Get started",
+    url: "/getting-started",
+  },
+};
+
+const ctaContent = {
```
Comment (usulpro) [inline docs/src/content/home.ts]: we don't have a cta separate block, so if any component includes CTA inside - it should have a field for that

File: docs/src/content/home.ts [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+    url: "/getting-started",
+  },
+};
+
+const vibrantCommunityBlockContent = {
+  title: "Vibrant community",
+  text: "Great tech attracts great people. Join the Hardhat community to find answers to your problems and contribute to the plugin ecosystem.",
+  cta: {
+    title: "Join the Hardhat Discord",
+    // TODO: switch to page reference later
```
Comment (usulpro) [inline docs/src/content/home.ts]: please update this part. we already have this page so we can use <Link> to navigate relatively

File: docs/src/content/home.ts [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+      text: "Keep your momentum going by making your development feedback loop up to 10x faster with Hardhat.",
+    },
+    articleTwo: {
+      title: "TypeScript",
+      text: "Catch mistakes before you even run your code by switching to a typed language. Hardhat provides full native support for TypeScript.",
+    },
+  },
+};
+
+const bannerContent = {
```
Comment (usulpro) [inline docs/src/content/home.ts]: Banner is also common so please move it to the config

File: docs/src/components/landingBlocks/BuiltByBlock.tsx [hunk 7]
```diff
@@ -5,10 +5,10 @@ import NomicFoundationLogo from "../../assets/images/nomic-foundation-logo.png";
 
 import Section from "../Section";
 import { appTheme } from "../../themes";
-import defaultProps from "../ui/default-props";
+import homepageContent from "../../content/home";
 
 interface Props {
-  content: typeof defaultProps.defaultBuiltByBlockContent;
+  content: typeof homepageContent.builtByBlockContent;
```
Comment (usulpro) [inline docs/src/components/landingBlocks/BuiltByBlock.tsx:11]: are we importing `homepageContent` only to get a type? let's switch to a explicit type declaration. Not in this PR. Please create a task in Trello

**Review by usulpro — CHANGES_REQUESTED** [review]
Please add Storybook

File: docs/src/content/home.ts (hunk 1 above)
Comment (Kragov) [reply docs/src/content/home.ts]: Done

**Pull request merged**